### DISCUSSION TITLE
Too many mod "hover" sounds prevent "toggle" one from playing

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#27618] [Q&A]
The video is clearly more of a "stress test" example, but I've often noticed this problem without understanding why it happens, and I've come to the conclusion that it seems to be related to too many(?) hover sounds.

Simply put, hovering a lot of mods in a short period of time prevents the "click" sound of a mod you are toggling.
It may be that this works 100% as intended, so please let me know.

https://github.com/ppy/osu/assets/43073074/83f2bf2b-8ad3-47f1-8f1d-f86d699e845e

**Comment by Gabixel** [comment]
Like I said, this is probably intended (some kind of limited number of samples?), but I noticed that this can usually happen by accident and confused me at first. Maybe it's something worth changing/keeping in mind? (At the moment I have no other scenarios in mind where something like this could easily happen.)

**Reply by Joehuu** [reply]
Because of this:
https://github.com/ppy/osu/blob/9ba4bf5fb74bff7f185d628b1db77e490401fb12/osu.Game/Overlays/Mods/ModSelectPanel.cs#L180
https://github.com/ppy/osu/blob/9ba4bf5fb74bff7f185d628b1db77e490401fb12/osu.Game/Overlays/Mods/ModSelectPanel.cs#L206-L216

The toggle sample is dependent on `Static.LastHoverSoundPlaybackTime` (which seems like incorrect usage). It's there because presets trigger multiple panel toggle samples at once. Should probably make a separate `LastClickSoundPlaybackTime`.
